components/test-project/src/main/resources/META-INF/dirigible/test-project/openapi/test.openapi

```diff
@@ -0,0 +1,34 @@
+openapi: 3.0.1
+info:
+  title: Eclipse Dirigible
+  description: Eclipse Dirigible API of the REST services provided by the applications.
+  termsOfService: https://www.dirigible.io/
+  contact:
+    name: Eclipse Dirigible
+    url: https://www.dirigible.io/
+    email: dirigible-dev@eclipse.org
+  license:
+    name: Eclipse Public License - v 2.0
+    url: https://www.eclipse.org/legal/epl-v20.html
+  version: 8.0.0
+servers:
+- url: /services/js
+paths:
+  /test-project/openapi/api.mjs/hello-world:
```

Comment:
@delchev, @vmutafov

To align which format to be used: 

**Option 1:**
----

```yaml
servers:
- url: /
paths:
  /services/js/test-project/openapi/api.mjs/hello-world
```

<img width="1470" alt="option1" src="https://github.com/eclipse/dirigible/assets/4092083/8317d540-96de-40b9-ab26-a0ac227a6bbc">

- **Pros**:
  - The most explicit way to specify the paths.
  - The `servers` could be used to specify different domains/instances, to test the OpenAPI.
  - Doesn't affect the `Open API` view and the `Preview` view.
- **Cons**:
  - Too long paths.

**Option 2:**
----

```yaml
servers:
- url: /services/js/test-project
paths:
  /openapi/api.mjs/hello-world:
```

<img width="1492" alt="option2" src="https://github.com/eclipse/dirigible/assets/4092083/47bbea76-f209-4366-ad7a-912f006271fd">

- **Pros**:
  - The most simplified way to specify the paths _(contains only the path **inside** the project)_.
  - It appears best in the "Preview" view.
- **Cons**:
  - It would be complicated inside the `OpenAPI` view to determine which paths belongs to which project and respectively to which `server`.

**Option 3:**
----
```yaml
servers:
- url: /services/js
paths:
  /test-project/openapi/api.mjs/hello-world:
```

<img width="1470" alt="option3" src="https://github.com/eclipse/dirigible/assets/4092083/2f77b25d-a1ca-4bcd-ba06-09efd860d6f0">

- **Pros**:
  - The middle between being too explicit and relatively simplified way to specify the paths _(contains the path **including** the project name)_.
  - Inside the `OpenAPI` view it would be possible to determine which path to which project belongs and to execute services, as we'll have only one server `services/js`.
- **Cons**:
  - It would be complicated inside the `OpenAPI` view to determine which paths belongs to which project and respectively to which `server`.
  - Also it would be harder to execute services, as first it has to be determined which path to which server belongs.

> _**Note:** The current version of the PR is implementing `Option 3`_

Reply:
Option 2 looks great to me!